pandora-client-web/src/common/useTextFormatting.ts

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+export function useTextFormatting(originalRef: React.MutableRefObject<HTMLTextAreaElement> | React.ForwardedRef<HTMLTextAreaElement>): React.RefObject<HTMLTextAreaElement> {
```

Comment:
I am not sure the name describes the function well:
Maybe `useTextFormattingOnKeyboardEvent`?